### DISCUSSION TITLE
SPlisHSPlasH & DFSPH in two cool applications

### DISCUSSION THREAD

**Discussion by mambobanda** [InteractiveComputerGraphics/SPlisHSPlasH#147] [Show and tell]
Hi,

This repository has helped my work so much. I thought I would take the time to share what I have accomplished with it. I also have to personally thank Jan Bender, because before SPlisHSPlasH existed he took the time to answer and verify some of my code when I was implementing DFSPH.

SPlisHSplasH has been integrated into a popular Lightwave plugin called Deep Rising FX (https://www.deepfxworld.com/deep-rising-fx.php). 

My own implementation of DFSPH (using SPlishSPlash as a reference) is part of the physics based animation package Deep FX Studio (https://www.deepfxworld.com/deep_fx_studio.php).

Even though SPlishSPlash is targeted at the scientific community, I have mainly used it for producing images & video for visual consumption.

https://user-images.githubusercontent.com/18059953/145221690-5c3b37b5-2b25-4a8d-aeed-c0bd3574635b.mp4

**Comment by janbender** [comment]
Looks nice! Thanks for sharing that.
